automated/bin/Archive_Concordance.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# identify on which cluster this script will be executed
+HOST=$(hostname -s)
+
+# source config files
+
+source ${EBROOTGAP}/automated/etc/${HOST}.cfg
+
+#Checking if VCF File array is older then ArchiveTime and archive file if this is the case
+
+for i in $(find "/groups/umcg-gd/${TMP_LFS}/Concordance/array/" -type f -maxdepth 1 -mtime +31 -print)
```

Comment:
what is this -print doing?

Reply:
What it should do and what you can lookup in the man pages for the ```find``` command ;)
This is the better and safer option compared to an ```ls```.